t/jobspec/valid/basic.yaml~

```diff
@@ -0,0 +1,19 @@
+version: 1
```

Comment:
Is this backup file included by accident?

Reply:
Yes, thanks!